Header header
std_msgs/Bool status
float32 xPos
float32 yPos